src/components/Markdown/Tiles.tsx

```diff
@@ -0,0 +1,36 @@
+import Link from 'src/components/Link';
+import Icon from '@ably/ui/core/Icon';
+import { IconName } from '@ably/ui/core/Icon/types';
+
+export type TileProps = {
+  title: string;
+  description: string;
+  image: IconName;
+  link: string;
+};
+
+export const Tile = ({ title, description, image, link }: TileProps) => {
+  return (
+    <Link to={link} rel="noreferrer" className="block no-underline">
+      <section className="flex items-start gap-4 p-4 bg-neutral-100 rounded-lg border border-neutral-300 transition-colors hover:bg-neutral-200 hover:border-neutral-400 min-h-[106px]">
+        <Icon name={image} size="2rem" />
+        <div className="flex-1 max-w-[264px]">
+          <h3 className="text-[18px] font-bold leading-[120%] tracking-tight mb-2 text-neutral-1300">{title}</h3>
```

Comment:
`text-lg` [maps to 18px](https://v3.tailwindcss.com/docs/font-size) - we should aim to use standard classes over arbitrary classes (the ones in square brackets) wherever possible